integration-tests/cli/csv-parser-order.it.js

```diff
@@ -0,0 +1,183 @@
+import { exec } from 'child_process'
+import fs from 'fs'
+import path from 'path'
+import streamtest from 'streamtest'
+import tmp from 'tmp'
+import {
+  AddReturnInfoCsvParser,
+  LineItemStateCsvParser,
+} from '../../packages/csv-parser-order/src/index'
+import { version } from '../../packages/csv-parser-order/package.json'
+
+describe('CSV and CLI Tests', () => {
+  const binPath = './integration-tests/node_modules/.bin/csvparserorder'
+  const samplesFolder = './packages/csv-parser-order/test/data/'
```

Comment:
maybe we remove the trailing `/` here, makes the variable looks more like a path